package-diff

```diff
@@ -60,6 +60,50 @@ fi
 curl --location --silent -S -o "$A" "$URL_A"
 curl --location --silent -S -o "$B" "$URL_B"
 
+function no_hardlinks {
+    local f="${1}"; shift
+    local -a lines
+    lines=()
+    local line pair clear_line hardlink size key
+    # cache contains entries in form of <hardlink_count>:<size>, if
```

Comment:
Since this is ambiguous the result for Alpha is now just 674 MB instead of 724 MB shown by `df`. We could change the contents text file to also include the inode, or we now hit the point where we should generate a new text file with `df` output.

Reply:
As a follow-up work, I'll experiment with generating another contents file with inode numbers in it. And probably without timestamps and other data we don't seem to need.